avm/ptn/azd/ml-project/main.bicep

```diff
@@ -0,0 +1,171 @@
+metadata name = 'Machine Learning workspace'
```

Comment:
Ca we please specify Azd in the readme title to avoid any confusion with the resource module?